test/transforms/dec128.test.js

```diff
@@ -22,52 +24,101 @@ const doesNotChangeNumbers = {
     "(1 + (4 - 2)) * (3 + 4);",
 };
 
+const constructor = {
+  "works with plain number": {
+    code: "Decimal(1);",
+    output: `${libName}(1);`
+  },
+  "works with string": {
+    code: 'Decimal("1");',
+    output: `${libName}("1");`
+  },
+  "passes expression through": {
+    code: 'Decimal(1 + 1);',
+    output: `${libName}(1 + 1);`
+  },
+  "throws on null": {
+    code: "Decimal(null)",
+    error: "TypeError: Can't convert null or undefined to Decimal."
+  },
+  "throws on explicit undefined": {
+    code: "Decimal(undefined)",
+    error: "TypeError: Can't convert null or undefined to Decimal."
+  },
+  "throws on implicit undefined": {
+    code: "Decimal()",
+    error: "TypeError: Can't convert null or undefined to Decimal."
+  },
```

Comment:
I'd add a "throws when called with new" test here. The proposal doesn't define what happens when you call `new Decimal(...)`, but `new BigInt(...)` throws "TypeError: BigInt is not a constructor" so I'm assuming we'd need to do the same.

Although that probably means writing some more code to visit NewExpression in the transforms. Given that, it also seems like it should be fine to kick that down the road.

Reply:
I've added this with a link back to this discussoin to the to do list. Nice catch.